Frontend updates - July 2021 (#504)
SenseNet/sncom-resources: _updates/2021-07-18-july.md

---
title: July 2021
description: sensenet frontend updates of July 2021
---

# July 2021

Summer, beach, vacation 🍸🌴☀️

![Beach](/img/updates/beach.jpg)
_Photo by [Artiom Vallat]("https://unsplash.com/@virussinside") on Unsplash_

> Summer is usually about holiday and relaxation. Of course, we are not left out of the good either, but we have not stopped working completely. Here are the changes we did:

Reviewer: ```suggestion
Summer is usually about holiday and relaxation. We are finally able to breathe more freely after a year of quarantine and isolation, but we have not stopped working completely. Here are the features we added and changes we made:
```

## sn-editor-react@0.2.0

> Our richtext editor uses lazy-loading from now: [1355](https://github.com/SenseNet/sn-client/pull/1355)

Reviewer: ```suggestion
Our richtext editor uses lazy-loading from now on: [1355](https://github.com/SenseNet/sn-client/pull/1355)
```

> It works with localiztaion resources (english and hungarian is available) [1351](https://github.com/SenseNet/sn-client/pull/1351)

Reviewer: ```suggestion
It uses localized string resources (English and Hungarian languages are available) [1351](https://github.com/SenseNet/sn-client/pull/1351)
```

Has a new feature: table insertion and editing 🎉 [1371](https://github.com/SenseNet/sn-client/pull/1371)

![Editor table](/img/updates/editor-table.gif "Editor table")

## gatsby-source-sensenet@0.1.1 and @sn-blog-gatsby

> We have removed the sn-blog-gatsby from our monorepo and placed to an independent github repository as [gatsby-starter-sensenet-blog](https://github.com/SenseNet/gatsby-starter-sensenet-blog). We had some bugfixes on the source plugin as well. [1367](https://github.com/SenseNet/sn-client/pull/1367)

Reviewer: ```suggestion
We have moved the sn-blog-gatsby component from our monorepo to an independent github repository as [gatsby-starter-sensenet-blog](https://github.com/SenseNet/gatsby-starter-sensenet-blog). We made some bugfixes on the source plugin as well. [1367](https://github.com/SenseNet/sn-client/pull/1367)
```

> ❗❗ **We also have really sad news**: 2 of them left the team in this month. 😢🎈

Reviewer: ```suggestion
❗❗ **We also have really sad news**: two of our members left the team this month. 😢🎈
```
